bokehjs/src/less/buttons.less

```diff
@@ -10,10 +10,12 @@
 
   padding: var(--padding-vertical) var(--padding-horizontal);
   font-size: var(--font-size);
-  border: 1px solid transparent;
+  border: var(--border-width) var(--border-style) transparent;
   border-radius: var(--border-radius);
 
   outline: 0;
+  // @mattpap - I don't really like this style of inset focus ring
+  // but I guess this should be tackled in a separate PR?
   outline-offset: -5px;
```

Comment:
Yes, as we discussed in the meeting. In fact, please start an issue for this, so that we have a place to discuss possible improvements.